Procfile-dev

```diff
@@ -1,5 +1,3 @@
-republik: cd servers/republik && PORT=5000 yarn run dev
-republik-scheduler: cd servers/republik && PORT=5000 yarn run devScheduler
-publikator: cd servers/publikator && PORT=5010 yarn run dev
-publikator-scheduler: cd servers/publikator && PORT=5010 yarn run devScheduler
+graphql: cd servers/graphql && PORT=5000 yarn run dev
+scheduler: cd servers/graphql && PORT=5000 yarn run devScheduler
```

Comment:
Taken that graphql and scheduler both run on port 5000, aren't these going to conflict?

Reply:
The scheduler actually doesn't listen on any port, I removed the var.